ConsumerApi/RelationshipsDbContextSeeder.cs

```diff
@@ -64,4 +68,22 @@ private async Task FillContentColumnsFromBlobStorage(RelationshipsDbContext cont
             await context.SaveChangesAsync();
         }
     }
+
+    private async Task FillRelationshipTemplatesWithMissingContent(RelationshipsDbContext context)
+    {
+        var relationshipTemplatesWithMissingContent = await context.RelationshipTemplates
+            .Where(rt => rt.Content == null).ToListAsync();
+
+        foreach (var relationshipTemplate in relationshipTemplatesWithMissingContent)
+        {
+            if (relationshipTemplate.Content == null)
```

Comment:
Why do you check if the content is null? You only selected those templates which don't have a content.

Reply:
Stupid mistake :/